wsgi.py

```diff
@@ -37,10 +37,16 @@
 from thoth.storages import __version__ as thoth_storages_version
 
 
-__version__ = "0.6.1" + "+thoth_storage." + thoth_storages_version
+__version__ = (
+    "0.7.1"
+    + "+thoth_storage."
+    + thoth_storages_version
+    + "+thoth_common."
+    + thoth_common_version
+)
 
 
-init_logging()
+init_logging(logging_env_var_start="THOTH_RESULT_API_LOG_")
```

Comment:
It would be better to patch the line bellow to be consistent with other services:

```python
application = Flask("thoth.result_api")
```